src/Search.tsx

```diff
@@ -0,0 +1,123 @@
+import 'swiper/swiper.min.css';
+import "swiper/css";
+import {Swiper, SwiperSlide} from "swiper/react"
+import { SlArrowLeft, SlArrowRight } from 'react-icons/sl'
+import { useEffect, useState } from "react"
+import SwiperCore, { Autoplay } from 'swiper';
+import { useSelector, useDispatch } from "react-redux";
+import { SearchState } from './types/type';
+import { Link } from 'react-router-dom';
+import { searching } from './features/searchSlice';
+import { SearchNameState } from './types/type';
+import Error from './Error';
+import Loading from './components/Loading';
+import { NewsState } from './types/type';
+
+SwiperCore.use([Autoplay]);
+
+export default function Search(){
+  const News:NewsState[] = useSelector((prev: SearchState) => prev.search.value)
+  const [slide, setSlide] = useState<any>();
```

Comment:
Could you remove any type or specify the type the slide state will have?

Reply:
removing the type is resulting in an error and yet there is no way I can know the value of the slide